plugin/src/main/scala/com/lightbend/sbt/JavaFormatterPlugin.scala

```diff
@@ -93,12 +91,14 @@ object JavaFormatterPlugin extends AutoPlugin {
       includeFilter in format := "*.java",
       sourceLevel := None,
       targetLevel := None,
-      javaFormattingSettingsFile := None,
-      javaFormattingSettingsFile := javaFormattingSettingsFile.value,
+      javaFormattingSettingsFile in ThisBuild := Some(baseDirectory.value / "formatting-java.xml"),
```

Comment:
This was me being confused about the old plugin code, and our use of it in the cinnamon build. Sorry about that.